File: java/doit_java/src/chap05/standalone/Order.java
/*
나혼자 코딩!
2. 쇼핑몰에 주문이 들어왔습니다. 주문 내용은 다음과 같습니다.

 주문 번호: 202011200001
 주문자 아이디: anifilm
 주문 날짜: 2020년 11월 20일
 주문자 이름: 임채영
 주문 상품 번호: PD0345-12
 배송 주소: 서울시 중랑구 동일로 157길 20-7

위 주문 내용에 대한 클래스를 만들고 주문 내용을 인스턴스로 생성한 후 위와 같은 형식으로
주문 내용을 그대로 출력해 보세요.
 */
package chap05.standalone;

public class Order {
    long orderId;
    String customerId;
    String orderDate;
    String customerName;
    String productId;
    String shippingAddress;
}

File: python/p_oop2/chap05/excercise/order.py
class Order:
    # 멤버변수 선언
    order_id = None
    customer_id = None
    order_date = None
    customer_name = None
    product_id = None
    shipping_address = None


def main():
    order = Order()

    order.order_id = 201803120001
    order.customer_id = "abc123";
    order.order_date = "2018년 3월 12일";
    order.customer_name = "홍길순";
    order.product_id = "PD345-12";
    order.shipping_address = "서울시 영등포구 여의도동 20번지";

    print("주문 번호:", order.order_id)
    print("주문자 아이디:", order.customer_id)
    print("주문 날짜:", order.order_date)
    print("주문자 이름:", order.customer_name)
    print("주문 상품 번호:", order.product_id)
    print("배송 주소:", order.shipping_address)


if __name__ == '__main__':
    main()
